modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormFieldTemplateContextFactory.java

```diff
@@ -559,16 +559,23 @@ protected void setDDMFormFieldTemplateContextValidation(
 			return;
 		}
 
+		LocalizedValue errorMessageLocalizedValue =
+			ddmFormFieldValidation.getErrorMessage();
+
+		String errorMessage = null;
```

Comment:
You should use the method `GetterUtil.getString`, as in the removed code, to set the `errorMessage` value and avoid a potential NPE.